packages/contracts-bedrock/scripts/DeployOPChain.s.sol

```diff
@@ -566,8 +567,9 @@ contract DeployOPChain is Script {
         require(address(messenger.portal()) == address(_doo.optimismPortalProxy()), "L1xDM-40");
         require(address(messenger.superchainConfig()) == address(_doi.opcmProxy().superchainConfig()), "L1xDM-50");
 
-        bytes32 xdmSenderSlot = vm.load(address(messenger), bytes32(uint256(204)));
-        require(address(uint160(uint256(xdmSenderSlot))) == Constants.DEFAULT_L2_SENDER, "L1xDM-60");
+        // TODO: vm.expectRevert is not supported by op-chain-ops, so we can't check this yet.
+        // vm.expectRevert("CrossDomainMessenger: xDomainMessageSender is not set");
+        // messenger.xDomainMessageSender();
```

Comment:
Add a link to the [issue here](https://github.com/ethereum-optimism/optimism/issues/12720).